turbopack/crates/turbo-tasks-backend/src/backend/operation/mod.rs

```diff
@@ -106,27 +103,6 @@ impl<'a> ExecuteContext<'a> {
             self.backend.operation_suspend_point(|| op.clone().into());
         }
     }
-
-    pub fn suspending_requested(&self) -> bool {
-        self.backend.suspending_requested()
-    }
-
-    pub fn run_operation(
-        &self,
-        parent_op_ref: &mut impl Operation,
-        run: impl FnOnce(ExecuteContext<'_>),
-    ) {
-        let parent_op = take(parent_op_ref);
-        let parent_op: AnyOperation = parent_op.into();
-        let inner_ctx = ExecuteContext {
-            backend: self.backend,
-            turbo_tasks: self.turbo_tasks,
-            _operation_guard: None,
-            parent: Some((&parent_op, self)),
-        };
-        run(inner_ctx);
-        *parent_op_ref = parent_op.try_into().unwrap();
-    }
```

Comment:
Removed code

Reply:
Yeah, this was intentional. Clippy flagged this as unused, and after removing all tests pass.

Reply:
cc @sokra